tensorflow/python/ops/script_ops.py

```diff
@@ -80,10 +80,14 @@ def __init__(self, func, Tout, is_grad_func):
         None.
       is_grad_func: Whether this EagerFunc is the gradient of another
         EagerPyFunc.
+      use_tape_cache: (Optional.) Whether to cache `func` in the `tape_cache`.
+        NOTE(lithuak): see the note for `_eager_py_func`.
```

Comment:
the `NOTE(...):` is tripping up lint checker. I suggest rewording this to:

For additional information, see description of `_eager_py_func`.